### PR TITLE
Handle pending petitions from Enrollee

### PR DESCRIPTION
**Pull request by lionick**
Database change

`alter table cm_co_settings add enable_pending_petition_handler boolean;`

### CODE REVIEW

File: app/Controller/CoPetitionsController.php [hunk 1]
```diff
@@ -726,6 +734,45 @@ protected function dispatch($step, $id=null) {
...
+            }
+            else if(!empty($petitions) && count($petitions) > 1) {
+              // Redirect to petitions index page
+              $petition_redirect = [
+                'controller' => 'co_petitions',
+                'plugin' => null,
+                'action' => 'index/',
+                'coef' => $this->request->params['named']['coef'],
+                'co' => 2,
+                'op' => 'resume'
```
Comment (ioigoume) [inline app/Controller/CoPetitionsController.php:766]: `resume` have to be an enumerator variable

File: app/Controller/CoPetitionsController.php [hunk 2]
```diff
@@ -2125,6 +2173,9 @@ function isAuthorized() {
                        && ($roles['admin'] || $roles['subadmin']))
                    || $this->Role->isApprover($roles['copersonid']));
     
+    if(!empty($this->params['named']['op']) && $this->params['named']['op'] == 'resume') {
```
Comment (ioigoume) [inline app/Controller/CoPetitionsController.php:2176]: `resume` : enumerator

**Review by ioigoume — CHANGES_REQUESTED** [review]

**Review by ioigoume — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**